src/extended_json.ts

```diff
@@ -380,3 +275,122 @@ function serializeDocument(doc: any, options: EJSONOptions) {
     throw new Error('_bsontype must be a string, but was: ' + typeof bsontype);
   }
 }
+
+/**
+ * EJSON parse / stringify API
+ * @public
+ */
+// the namespace here is used to emulate `export * as EJSON from '...'`
+// which as of now (sept 2020) api-extractor does not support
+// eslint-disable-next-line @typescript-eslint/no-namespace
+export namespace EJSON {
```

Comment:
As the comment states, the ideal code would be to use `export * as ESJON from './...'` in bson.ts but unfortunately that syntax is not supported by api-extractor yet, so this is an alternative that could be changed later without breaking changes.
Also unlike our previous solution (a `const EJSON = {...}` in bson.ts) this will thread the documentation through to the user's import, before it was being lost.